rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc

```diff
@@ -318,12 +318,16 @@ Type::
 
 | `protocol`
 | `string`
-| protocol specifies whether the IngressController expects incoming connections to use plain TCP or whether the IngressController expects PROXY protocol. 
- PROXY protocol can be used with load balancers that support it to communicate the source addresses of client connections when forwarding those connections to the IngressController.  Using PROXY protocol enables the IngressController to report those source addresses instead of reporting the load balancer's address in HTTP headers and logs.  Note that enabling PROXY protocol on the IngressController will cause connections to fail if you are not using a load balancer that uses PROXY protocol to forward connections to the IngressController.  See http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt for information about PROXY protocol. 
- The following values are valid for this field: 
- * The empty string. * "TCP". * "PROXY". 
+| protocol specifies whether the IngressController expects incoming connections to use plain TCP or whether the IngressController expects PROXY protocol.
+ PROXY protocol can be used with load balancers that support it to communicate the source addresses of client connections when forwarding those connections to the IngressController.  Using PROXY protocol enables the IngressController to report those source addresses instead of reporting the load balancer's address in HTTP headers and logs.  Note that enabling PROXY protocol on the IngressController will cause connections to fail if you are not using a load balancer that uses PROXY protocol to forward connections to the IngressController.  See http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt for information about PROXY protocol.
+ The following values are valid for this field:
+ * The empty string. * "TCP". * "PROXY".
  The empty string specifies the default, which is TCP without PROXY protocol.  Note that the default is subject to change.
 
+ | `hostNetwork`
+ | `integer`
+ | hostNetwork is used to set binding ports for the Ingress Controller so that multiple Ingress Controllers can be deployed on the same node. The following optional node ports with default values can be used; `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`. The range of the priority value can go from positive to negative in the range of int32.
```

Comment:
There's an extra sentence in here which is unrelated.
> The range of the priority value can go from positive to negative in the range of int32

Reply:
Removed